artemis-commons/src/main/java/org/apache/activemq/artemis/utils/actors/ProcessorBase.java

```diff
@@ -108,21 +103,21 @@ public void shutdown(long timeout, TimeUnit unit) {
     * It will wait the current execution (if there is one) to finish
     * but will not complete any further executions
     */
-   public List<T> shutdownNow() {
```

Comment:
I've changed the interface making use of a functional interface in order to avoid very long backlog of stopped tasks to kill the java heap.
The original behaviour could be implemented anyway:
```
final List<T> stoppedTasks = new ArrayList<>();
if (executor.shutdownNow(stoppedTasks::add)>0){
    //do something with the stopped tasks
}
```